lib/rspec/github/example_decorator.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'delegate'
```

Comment:
I was told once by an RSpec maintainer to specifically not require anything because it could lead to tests passing despite actual code not working because the actual code does not require anything.

Reply:
Hmm, good point... 

I hope you don't mind too much that I started changing your PR but I had this idea for quite a bit since it was getting more and more hard to read.